src/coreclr/src/inc/pedecoder.h

```diff
@@ -349,6 +351,9 @@ class PEDecoder
     BOOL HasNativeEntryPoint() const;
     void *GetNativeEntryPoint() const;
 
+    // Look up a named symbol in the export directory
+    uint32_t GetExport(LPCSTR exportName) const;
```

Comment:
Why not return the actual pointer to the export here? (And null if it does not exist.)

Reply:
Good point, changed in 16ht commit.